site/en/guides/border-router/_toc.yaml

```diff
@@ -6,6 +6,8 @@ toc:
   path: /guides/border-router/beaglebone-black
 - title: Raspberry Pi
   path: /guides/border-router/raspberry-pi
+- title: ESP Thread Border Router
```

Comment:
Let's use the vendor/platform name here, like the others.  For this one I'd do "Espressif ESP32".

Also, can you move this entry between BeagleBone Black and Raspberry Pi, please?  We list them alphabetically so it doesn't look like we're preferring one vendor or platform over another.

Reply:
@Vyrastas Both BeagleBone Black and Raspberry Pi are the platforms of the official Linux-based OTBR (ot-br-posix), but the new ESP Thread Border Router is FreeRTOS-based. In other words, the new "Espressif ESP32" is not actually a platform of ot-br-posix.

So the desired way is dividing the current platform list into two parts:
- OTBR (Linux-based)
  - BeagleBone Black
  - Raspberry Pi
- OTBR (FreeRTOS-based)
  - Espressif ESP32 Series

And then make it clear from this page: https://openthread.io/guides/border-router.

Reply:
@chshu Ah I wasn't aware of that, thanks for the clarification.  Yes let's do it that way then.  I'd propose this for the menu:

```
toc:
- title: Overview
  path: /guides/border-router/
- heading: Platforms - Linux
- title: BeagleBone Black
  path: /guides/border-router/beaglebone-black
- title: Raspberry Pi
  path: /guides/border-router/raspberry-pi
- heading: Platforms - FreeRTOS
- title: Espressif ESP32 Series
  path: /guides/border-router/espressif-esp32
```

...and the rest as is

We'll update the border-router index page separately since it's not available on GitHub.  We'll add a news item to the front page as well.